RNA-seq/setup/exercises/04-prep_medulloblastoma_data.Rmd

```diff
@@ -0,0 +1,110 @@
+---
+title: "Prep medulloblastoma data for daily exercise"
```

Comment:
I was not expecting to see this notebook included here?

Reply:
No, it should be removed. I added it originally for reference, but neglected deleting it!